lib/i18n/tasks/translators/openai_translator.rb

```diff
@@ -63,6 +63,8 @@ def translate_values(list, from:, to:, **options)
     end
 
     def translate(values, from, to)
+      # Prompt from
+      # https://github.com/ObservedObserver/chatgpt-i18n/blob/main/src/services/translate.ts
```

Comment:
That repo has AGPL license, i18n-tasks has MIT license.
Please try to come up with your own prompt.

Reply:
I simply removed the comment because I had already come up with a different version of the prompt. The comment was here to tell where I got the inspiration from.